bootstrap.sh

```diff
@@ -2,44 +2,47 @@
 
 set -e
 
-HOSTBIN=$(go env GOHOSTOS)_$(go env GOHOSTARCH)/bin
-export GOBIN=$(pwd)/$HOSTBIN
+export GOBIN=$(pwd)/$(go env GOHOSTOS)_$(go env GOHOSTARCH)/bin
```

Comment:
is this a regression? You might need to do a git pull --rebase, not sure.

Reply:
Oh that's weird: I had rebased and then `git checkout --theirs -- bootstrap.sh` but it didn't take the most recent changes.